README.md

```diff
@@ -204,11 +204,44 @@ These build targets are supported:
 | electron      | Builds the Electron binary **(Default)**                 |
 | electron:dist | Builds the Electron binary and generates a dist zip file |
 | mksnapshot    | Builds the `mksnapshot` binary                           |
-| node:headers  | Builds the node headers `.tar.gz` file                   |
+| electron:node_headers  | Builds the node headers `.tar.gz` file          |
+| electron:electron_symbols  | Generate the breakpad symbols in release builds     |
+
+To build a specific target, run `e build --target [target]`. Running `e build` with no
+target will build `electron` by default.
+
+Any extra args are passed along to [ninja][ninja], so for example
+`e build -v` runs a verbose build.
+
+To see an exhaustive list of all possible build targets, you can run `e d gn ls out/[TYPE]`,
+where `[TYPE]` is e.g. `Testing` or `Release` depending on your build type. This will log a long
+list of targets to the console and also allow you to build some of Chromium's targets.
```

Comment:
Worth noting that command must be run from `$root`, otherwise it will error.

Reply:
This isn't true anymore - i fixed that a bit back so `e build` can be run from anywhere and will run from `$root` on its own :D